source/retryable-reads/tests/gridfs-download-serverErrors.yml

```diff
@@ -171,3 +171,33 @@ tests:
         operations: [*retryable_operation_fails]
         expectations:
              - *retryable_command_started_event
+    -
+        description: "Download succeeds after ExceededTimeLimit"
+        failPoint:
+            <<: *failCommand_failPoint
+            data: { failCommands: [find], errorCode: 262 }
+        operations: [*retryable_operation]
+        expectations:
+             - *retryable_command_started_event
+             - *retryable_command_started_event
+             - *find_chunks_command_started_event
```

Comment:
Confirming that this is consistent with other "succeed" tests earlier in the file, which include a trailing `*find_chunks_command_started_event`.

Reply:
Yes all of the 'Download succeeds after ...' tests include a trailing *find_chunks_command_started_event*